src/components/AddChore/AddChore.jsx

```diff
@@ -13,7 +13,10 @@ function AddChore({ onAddChore }) {
 
   const submitButton = (e) => {
     e.preventDefault();
-    onAddChore({ ...chore });
+
+    if (!chore.choreName || !chore.dueDate) return;
+
+    onAddChore({ ...chore, id: Date.now() });
```

Comment:
I saw in later code, you're using this id to update/delete a specific chore. Are these all unique? As in does this include maybe milliseconds? Otherwise, I can see an issue arising if chores are created close to the same time.

Reply:
Date.now() does include milliseconds! I can see where your coming from regarding the issue of creating chores at the same time, but I feel like it's unlikely a user can create multiple chores within that time frame, but I can change it to something more unique down the line?

Reply:
Oh ok in that case, I think this is fine. I just wasn't sure how unique of an id this would give you, especially if it didn't include time and only date.